js/lesson_4/src/task.ts

```diff
@@ -22,13 +24,12 @@ export const users: PersonType[] = [
     {
         name: 'Batman',
         age: 'unknown',
-        isSuperHero: true,
```

Comment:
Why did you remove this property?
It is optional, same as 'occupation'.

Reply:
I´m already fixed